components/ui/Table.tsx

```diff
@@ -459,6 +464,12 @@ const Table = ({
                           ) : (
                             <></>
                           )}
+                          {column.infobox && (
+                            <InfoPopover
+                              title="Test"
```

Comment:
Think this should be adjusted to something meaningful or removed

Reply:
Sorry, forgot to remove this